docs/manual/creating-a-checker.tex

```diff
@@ -713,12 +709,11 @@
 --- a qualifier that is a (direct or indirect) subtype of every other
 qualifier.
 
-Your type system must give \<null> the bottom type.
+\<null> is the bottom type.
```

Comment:
A note about `java.lang.Void` would also be useful. In many of the places where you removed `@ImplicitFor` it was `@ImplicitFor(literals = {LiteralKind.NULL}, typeNames = {Void.class})`. Is this change implicitly applying bottom to both?

Reply:
I only made `null` bottom.  I think `Void` used to be the type of type variable lower bounds so it needed to be implicitly bottom.  But that's not true anymore.  

Is there some other reason for Void to be bottom? If so, I can add that reason here and make it bottom.

Reply:
The only value of the `Void` type is `null`. It is therefore most precise for both of them to have the same type. This has nothing to do with lower bounds. `Void` can also be used elsewhere.

For the Nullness Checker:

```java
  void foo(Void p) {
    p.toString();
  }
```

By making `Void` `@Nullable` one get's a useful error on the dereference.

Reply:
Thanks for the explanation.  I made this change.